### ISSUE TITLE
Pretrained Model

### ISSUE THREAD

**Issue by CoderHHX** [WangYueFt/rfs#8]
Hi, Yue! Will you release the pre-trained model for tieredImageNet, including use trianval and trian_only? I found to use the option --use_trainval the training processing seems weird that in the testing stage in every 100 episodes, the accuracy is 0.1-0.2. I use your code trian_supervised. It performs well using MiniImageNet data. 

Thanks.

**Comment by zhouchunpong** [comment]
Hi bro， have you got the the pre-trained model for tieredImageNet ?
